src/main/scala/org/apache/spark/shuffle/compat/spark_3_0/UcxLocalDiskShuffleExecutorComponents.scala

```diff
@@ -22,14 +23,18 @@ class UcxLocalDiskShuffleExecutorComponents(sparkConf: SparkConf)
   private var blockResolver: UcxShuffleBlockResolver = _
 
   override def initializeExecutor(appId: String, execId: String, extraConfigs: util.Map[String, String]): Unit = {
+    logDebug("LEO UcxLocalDiskShuffleExecutorComponents initializeExecutor appId: " + appId + " execId: " + execId)
```

Comment:
Regarding all the added logs:
1. Choose appropriate log level
2. More descriptive log message.

Reply:
Maybe we can add in the future some @interface / annotation that will log automatically function names and arguments for cleaner code?

Reply:
We actually use a framework (log4j) that prints the function name.
Example:
log from NvkvWrapper.scala:61 is printed like this:
`dispatcher-event-loop-3 23/08/19 14:35:38:821 DEBUG NvkvWrapper: **LEO NvkvWrapper constructor cpu_mask 1**`
The bold part is the string passed to log4j, all the rest is added by log4j.
We can adapt the log messages.
Regarding arguments, Scala syntax for embedding arguments in a string is pretty easy and clean:
Example:
`verbosedNvkvLogDebug(s"LEO NvkvWrapper: packedNvkv nvkvCtx ${nvkvCtx} nvkvCtxSize ${nvkvCtxSize} bb ${UnsafeUtils.getAdress(nvkvRemoteReadBuffer)}")` 

You can find an example for a log file here:
`/hpc/mtr_scrap/users/ofarjon/spark/example_log4j.log`

Reply:
After checking the logs again log4j print the Object name not function. [I think You can print the function name](https://logging.apache.org/log4j/1.2/apidocs/org/apache/log4j/PatternLayout.html). need to check it.

Reply:
I meant using something like AspectJ or some [library using AspectJ](https://aspects.jcabi.com/ ), and then functions would look like this:

```scala
@Loggable(DEBUG)
override def initializeExecutor(...) {
}
```
Which will print class name and function without the need to type it ourselves in every function

Reply:
Also:
![image](https://github.com/gleon99/sparkucx/assets/19361875/f36bd6ac-c185-4704-bc67-179e9d217572)

Reply:
Let's consider the possibility of replacing the logging library as a separate, stand-alone task - without blocking merging of this PR.